bedwars-plugin/src/main/java/com/tomkeuper/bedwars/listeners/InvisibilityPotionListener.java

```diff
@@ -41,6 +50,47 @@
  * potion or when the potion is gone. It is required because it is related to scoreboards.
  */
 public class InvisibilityPotionListener implements Listener {
+    private final List<Player> invisiblePlayers = new ArrayList<>();
+    private final boolean footstepsEnabled = BedWars.getAPI().getConfigs().getMainConfig().getBoolean(ConfigPath.GENERAL_CONFIGURATION_ENABLE_FOOTSTEPS_ON_INVISIBILITY);
```

Comment:
maybe you can use the method BedWars#config

Reply:
BedWars.config.getBoolean(ConfigPath.GENERAL_CONFIGURATION_ENABLE_FOOTSTEPS_ON_INVISIBILITY)

Reply:
Oh i didnt know there was that, thanks, ill change it rn